app/build.gradle

```diff
@@ -114,6 +114,14 @@ dependencies {
   // Logging
   implementation dependenciesList.timber
 
+  // Network
+  implementation(dependenciesList.okhttp) {
+    force = true
+  }
+  implementation(dependenciesList.okhttpInterceptor) {
+    force = true
+  }
+
```

Comment:
Why do we need to add this explicitly in the `app` test app but not in the `examples` one? Isn't that applied transitively?